### PR TITLE
PRO-1632 Replace Ramp with Onramper

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1632/replace-ramp-with-onramper

## Description

- Replaced Ramp with Onramper

## Motivation and Context

-

## How Has This Been Tested?

- Tested on Android device

## Screenshots (if appropriate):
https://github.com/pillarwallet/pillarwallet/assets/82652040/ca57b788-8de5-42c1-8ebc-140fbc2b8804

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by Yash-Chaubal — APPROVED** [review]
LGTM! Just wondering if we should set a default fiat param?

**Comment by lalitkumarrock** [comment]
> LGTM! Just wondering if we should set a default fiat param?

Actually, the fiat value is not mentioned in ticket.

File: src/locales/en/common.json [hunk 1]
```diff
@@ -1239,9 +1239,9 @@
           "highDeploymentFees": "Warning: High deployment fees if buying on Ethereum Mainnet",
           "deploying": "Deploying...",
           "options": {
-            "rampTitle": "RAMP Network",
-            "rampCountry": "US & Europe",
-            "rampDescription": "Buy with USD, EUR or GBP with Apple & Google Pay or Debit card. Cashing out from crypto available.",
+            "rampTitle": "Onramper",
+            "rampCountry": "Global",
+            "rampDescription": "Get the best rate from 18 supported Onramp providers with every preferred payment method and currency.",
```
[the remark this answers is not in the document]
Comment (lalitkumarrock) [reply src/locales/en/common.json]: Updated it!

File: src/configs/rampConfig.js [hunk 2]
```diff
@@ -41,4 +41,7 @@ export const ETHERSPOT_RAMP_CURRENCY_TOKENS = [
   OPTIMISM_DAI,
   OPTIMISM_ETH,
 ];
+
 export const ARCHANOVA_RAMP_CURRENCY_TOKENS = ETH;
+
+export const ONRAMPERSUPPORTEDCHAIN = [ETHEREUM, POLYGON, ARBITRUM, GNOSIS, BSC];
```
[the remark this answers is not in the document]
Comment (lalitkumarrock) [reply src/configs/rampConfig.js]: Updated it!

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**